x/feemarket/keeper/params.go

```diff
@@ -16,3 +17,24 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// ----------------------------------------------------------------------------
+// Parent Base Fee
+// Required by EIP1559 base fee calculation.
+// ----------------------------------------------------------------------------
+
+// GetConstantFee get's the base fee from the paramSpace
+// return nil if base fee is not enabled
+func (k Keeper) GetBaseFee(ctx sdk.Context) *big.Int {
+	params := k.GetParams(ctx)
+	if params.NoBaseFee {
+		return nil
+	}
```

Comment:
Should we also check `EnableHeight` here?

Reply:
hm not sure 🤔 

Those two parameters seems redundant. 

My understanding is `EnableHeight` controls the calculation while `NoBaseFee` controls the basefee as a whole